x-pack/plugins/transform/public/app/services/es_index_service.ts

```diff
@@ -7,11 +7,13 @@
 
 import type { HttpSetup } from '@kbn/core/public';
 import type { DataViewsContract } from '@kbn/data-views-plugin/public';
-import { API_BASE_PATH } from '../../../common/constants';
+import { addInternalBasePath } from '../../../common/constants';
 
 export class IndexService {
   async canDeleteIndex(http: HttpSetup) {
-    const privilege = await http.get<{ hasAllPrivileges: boolean }>(`${API_BASE_PATH}privileges`);
+    const privilege = await http.get<{ hasAllPrivileges: boolean }>(
+      addInternalBasePath(`privileges`)
```

Comment:
Do we need to add version here?

Reply:
updated in https://github.com/elastic/kibana/pull/158273/commits/33cfe9ee618d0cfdbf775abacf7bc8affba8693c